settings.gradle

```diff
@@ -7,5 +7,6 @@ include 'governator-commons-cli'
 include 'governator-test-junit'
 include 'governator-archaius'
 include 'governator-jetty'
+include 'governator-jersey'
```

Comment:
jersey1? Not sure if jersey2 actually requires us to implement anything similar at this point.